src/app/components/home/home.component.html

```diff
@@ -10,39 +10,38 @@
           </mat-card-header>
           <mat-card-content>
             <h1 class="cardColour text-center">Welcome to IIIT Rosei</h1>
-            <p class="text-center">This app is made by contributors of <a href="https://github.com/p-society/" target="_blank" mat-button color="accent">P-society</a>
-              and would request you to follow some guidelines listed below.</p>
+            <p class="text-center">This app is made by the contributors at <a href="https://github.com/p-society/" target="_blank" mat-button color="accent">P-society</a>
+              and we request you to kindly follow the guidelines listed below.</p>
 
                 <span class="listColur">
                   <p>
-                    This site is hosted on glitch and being a free service, it takes time of about 1 minute to wake the application if no traffic is there on server.
+                    This site is hosted on glitch and being a free service, it takes about 1 minute to wake the application up if there is no traffic on the server.
                   </p>
                 </span>
 
                 <span class="listColur">
                   <p>
-                    A lot of effort is put to make this application, so please don't act smart and try to register with other than your college ID.
+                    A lot of effort has been put into making this application, so please don't register with an ID that isn't your college ID.
                   </p>
                 </span>
 
                 <span class="listColur">
                   <p>
-                    Some people try to act smart and mess with database so if your account is not activated by 12:00 am(midnight), irrespective of your registeration
-                    then it will be automatically deleted.
+                    If your account is not activated by midnight, irrespective of your registration,
+                    then it will automatically be deleted. This step is being taken to prevent any tampering with the database.
                   </p>
                 </span>
 
                 <span class="listColur">
                   <p>
-                    We do not have any funds but still for your convenience we have hosted it on a server and using
-                    premium email service to provide better user experience in case you forget your password or didn't recieve an activation email.
+                    Despite non availability of funds, the app is being hosted on a server and is using premium email service to provide a better user experience, in case you forget your password or do not recieve an activation email.
```

Comment:
This sentence is long and ambiguous. Try to make it better by breaking it down.

Reply:
I don't understand what it means. Can you help me in that please?